### PR TITLE
incentives + distribution script

### PR DESCRIPTION
**Pull request by NimrodHunter**

### CODE REVIEW

File: script/IncentiveDistributor.s.sol [hunk 1]
```diff
@@ -2,39 +2,54 @@
...
+
+    // The amount of allocation points assigned to the pool. Also known as the amount of ARB to distribute per block.
+    uint256 glpAllocPoint = 3939; // 39,39% GLP
+    uint256 usdcAllocPoint = 3939; // 39,39% USDC
+    uint256 auraAllocPoint = 2122; // 21.22 % jAURA
+
+    address public incentiveReceiver; // Update
+
+    uint256 glpWithdrawIncentives; // Update
+    uint256 usdcWithdrawIncentives; // Update
```
Comment (0x-ultra) [inline script/IncentiveDistributor.s.sol:37]: 2% jGLP, 1% jUSDC

File: src/Sushi/MiniChefV2.sol [hunk 2]
```diff
@@ -110,20 +129,25 @@ contract MiniChefV2 is BoringOwnable, BoringBatchable {
...
     /// @param _deadline The timestmap for rewards deadline.
-    function setDeadline(uint256 _deadline) public onlyOwner {
+    function setDeadline(uint256 _deadline) public onlyGovernorOrOperator {
         deadline = _deadline;
     }
 
     /// @notice Set the `migrator` contract. Can only be called by the owner.
     /// @param _migrator The contract address to set.
-    function setMigrator(IMigratorChef _migrator) public onlyOwner {
+    function setMigrator(IMigratorChef _migrator) public onlyOperator {
```
Comment (0x-ultra) [inline src/Sushi/MiniChefV2.sol:150]: Sounds like a governor function?

File: script/IncentiveDistributor.s.sol (hunk 1 above)
Comment (NimrodHunter) [reply script/IncentiveDistributor.s.sol]: done

**Review by 0x-ultra — CHANGES_REQUESTED** [review]
Every admin function: only governor
Update pool weights: operator and governor

File: src/Sushi/MiniChefV2.sol (hunk 2 above)
Comment (NimrodHunter) [reply src/Sushi/MiniChefV2.sol:150]: let me see 👀

**Pull request merged**